public/talos/v1.11/platform-specific-installations/boot-assets.mdx

```diff
@@ -319,77 +345,107 @@ First, let's lookup extension images for Intel CPU microcode updates and `gvisor
 
 <Tabs>
   <Tab title="Command">
-    <CodeBlock lang="sh">
-      {`
-      crane export ghcr.io/siderolabs/extensions:${release_v1_11} | tar x -O image-digests | grep -E 'gvisor|intel-ucode'
-      `}
-    </CodeBlock>
+<CodeBlock lang="sh">
+{`
+crane export ghcr.io/siderolabs/extensions:${release_v1_11} | \\
+tar x -O image-digests | \\
+grep -E 'gvisor|intel-ucode'
+`}
+</CodeBlock>
   </Tab>
   <Tab title="Output">
-    <CodeBlock lang="sh">
-      {`
-        ghcr.io/siderolabs/gvisor:20231214.0-${release_v1_11}@sha256:548b2b121611424f6b1b6cfb72a1669421ffaf2f1560911c324a546c7cee655e
-    ghcr.io/siderolabs/intel-ucode:20231114@sha256:ea564094402b12a51045173c7523f276180d16af9c38755a894cf355d72c249d
-      `}
-    </CodeBlock>
+<CodeBlock lang="sh">
+{`
+ghcr.io/siderolabs/gvisor:20231214.0-${release_v1_11}@sha256:548b2b121611424f6b1b6cfb72a1669421ffaf2f1560911c324a546c7cee655e
+ghcr.io/siderolabs/intel-ucode:20231114@sha256:ea564094402b12a51045173c7523f276180d16af9c38755a894cf355d72c249d
+`}
+</CodeBlock>
   </Tab>
 </Tabs>
 
 Now we can generate the ISO image with the following command:
 
 <Tabs>
   <Tab title="Command">
-    <CodeBlock lang="sh">
-      {`
-        docker run --rm -t -v $PWD/_out:/out ghcr.io/siderolabs/imager:${release_v1_11} iso --system-extension-image ghcr.io/siderolabs/gvisor:20231214.0-${release_v1_11}@sha256:548b2b121611424f6b1b6cfb72a1669421ffaf2f1560911c324a546c7cee655e --system-extension-image ghcr.io/siderolabs/intel-ucode:20231114@sha256:ea564094402b12a51045173c7523f276180d16af9c38755a894cf355d72c249d --extra-kernel-arg net.ifnames=0 --extra-kernel-arg=-console --extra-kernel-arg=console=ttyS1
-        `}
-      </CodeBlock>
+<CodeBlock lang="sh">
+{`
+docker run --rm -t \\
```

Comment:
Do you know why the indents on this are so wide? The crane command is the same.

<img width="550" height="373" alt="image" src="https://github.com/user-attachments/assets/bb3dbce4-3b62-497d-bc22-f6fc678c1009" />

maybe @Slessi could help us figure out what the CSS is doing in this block